Sasha-svodnik/MJ-Yun/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,32 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+*/
+/* write comment!  */
+for (i=99; i>=0 ; i--){
+  if (i==1){
+    var line1= i + ' bottle of beer on the wall, ' + i + ' bottle of beer.'
+    var line2='Take one down and pass it around, no more bottles of beer on the wall.'
+  } else if(i==0){
+    var line1= 'No more bottle of beer on the wall, no more bottles of beer.'
+    var line2='Go to the store and buy some more, 99 bottles of beer on the wall.'
+  } else if(i==2) {
+    var line1= i + ' bottles of beer on the wall, ' + i + ' bottles of beer.'
+    var line2='Take one down and pass it around, '+ (i-1) + ' bottle of beer on the wall.'
+  } else{
+    var line1= i + ' bottls of beer on the wall, ' + i + ' bottles of beer.'
+    var line2='Take one down and pass it around, '+ (i-1) + ' bottles of beer on the wall.'
+  }
+
+  console.log(line1)
```

Comment:
Your code works just fine -- nicely done! From a performance perspective, it's best to avoid declaring the same variable multiple times. So one optimization you could make to this code would be to simply replace the `var` statements within each conditional statement with `console.log` statements.